Tools/adb_upload_to_bebop.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+BASEDIR=$(dirname $0)
```

Comment:
Can you use `BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`?
And below `SRC_DIR="$BASEDIR/../"`